static/styles/left-sidebar.scss

```diff
@@ -51,11 +57,6 @@ li.show-more-topics {
     }
 }
 
-#left-sidebar #user-list,
-#left-sidebar #group-pm-list {
-    padding-left: 10px;
-}
-
```

Comment:
Seems like you nested the `#left-sidebar` directly without reordering that makes the changes tough to read. It'd be better if you first commit the reordering CSS rulesets and later nest them in a separate commit like here [#14177](https://github.com/zulip/zulip/pull/14177). Same goes for other similar commits.

Reply:
I will make them in separate commits